types/tx_result.go

```diff
@@ -0,0 +1,14 @@
+package types
+
+import (
+	abci "github.com/tendermint/abci/types"
+)
+
+// TxResult contains results of executing the transaction.
+//
+// One usage is indexing transaction results.
+type TxResult struct {
+	Height    uint64                 `json:"height"`
```

Comment:
We decided to use `int` in places where it can go down (because otherwise it will require checks for underflow). But here `Height` only goes up, so it is OK.

`32` 4294967295 vs `64` 18446744073709551615 :)